fix #62

luca-vercelli opened the pull request:
This is essentially https://github.com/MycroftAI/mycroft-skills-manager/pull/63/commits/6adf346c984bd1aa330239757ae158afafdfd125, taking in account of given suggestions

luca-vercelli committed:
fix #62

This is essentially https://github.com/MycroftAI/mycroft-skills-manager/pull/63/commits/6adf346c984bd1aa330239757ae158afafdfd125, taking in account of given suggestions

forslund commented on msm/skill_state.py:
```diff
@@ -23,6 +24,12 @@ def load_device_skill_state() -> dict:
+    try:
+        # create folder if it does not exist
+        makedirs(dir_path)
+    except:
```
bare exceptions are generally frowned upon, make this `except Exception:`

forslund reviewed (changes requested):
Thanks, for contributing this fix. I flagged a minor caveat with the bare except. otherwise this looks good. To protect yourself and Mycroft we require you to sign a Contributor License Agreement (stating that you retain ownership of the contributed code but Mycroft is free to use it).

Please go to https://mycroft.ai/cla to start the process.

Once more thanks for picking up this issue!

luca-vercelli committed:
minor fix

As suggested by https://github.com/MycroftAI/mycroft-skills-manager/pull/70#discussion_r404848433

krisgesling commented:
CLA received and looks like the feedback has been addressed. Are you happy for this to get merged Ake?

forslund commented:
Sorry for the late review. it works fine, thank you for taking care of this.
